src/plugin/kubearmor.go

```diff
@@ -467,6 +467,10 @@ func StartKubeArmorRelay(StopChan chan struct{}, cfg types.ConfigKubeArmorRelay)
 					Owner:             &owner,
 				}
 
+				if strings.Contains(kubearmorLog.Labels, types.JobLabelControllerUidStr) {
+					kubearmorLog.Labels = libs.RemoveFieldFromLabel(kubearmorLog.Labels, types.JobLabelControllerUidStr)
+				}
```

Comment:
Both `Contains()` and `RemoveFieldFromLabel()` are going to iterate through entire label list. We can call `RemoveFieldFromLabel()` alone. It is sufficient.
```suggestion
				kubearmorLog.Labels = libs.RemoveFieldFromLabel(kubearmorLog.Labels, types.JobLabelControllerUidStr)
				}
```